server/gql/models/limitAndOffset/index.js

```diff
@@ -0,0 +1,12 @@
+import { GraphQLInt, GraphQLNonNull } from 'graphql';
+
```

Comment:
i know timestamps is also part of the models folder, but let's move this file and timestamps out of the gql/models directory. 


Let's create another directory for gql/fields with both of these in them.

Reply:
Sure.